packages/terra-docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Docs
+
+[![NPM version](https://badgen.net/npm/v/terra-docs)](https://www.npmjs.com/package/terra-docs)
+[![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.com/cerner/terra-core)
+
+{insert description}
```

Comment:
Need description

Reply:
coming soon, when content is actually added.